util/gdas_init/set_fixed_files.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [ "${FRAC_ORO:-"no"}" = "yes" ]; then
+  if  [ ${CTAR} == 'C48' ] ; then
+    OCNRES='400'
```

Comment:
For C48, I believe OCNRES should be 500.

Reply:
Good catch! It should be 500. I have fixed it.